test/modules/export/components/dialog/ExportDialogContent.test.js

```diff
@@ -39,7 +39,8 @@ describe('ExportDialogContent', () => {
 			})
 			.registerSingleton('TranslationService', { translate: (key) => key })
 			.registerSingleton('ProjectionService', projectionServiceMock)
-			.registerSingleton('FileSaveService', { saveAs: () => {} });
+			.registerSingleton('FileSaveService', { saveAs: () => {} })
+			.registerSingleton('ShareService', { copyToClipboard: () => {} });
```

Comment:
No need to register the dependencies of the `ExportItem` component, when we remove 
https://github.com/ldbv-by/bav4-nomigration/blob/3fe5b8b4721d51251a4ca888042be97b4b2eaed7/test/modules/export/components/dialog/ExportDialogContent.test.js#L10